pkg/chain/ethereum/beacon.go

```diff
@@ -147,6 +147,32 @@ func (bc *BeaconChain) OperatorToStakingProvider() (chain.Address, bool, error)
 	return chain.Address(stakingProvider.Hex()), true, nil
 }
 
+// OperatorToStakingProvider returns the staking provider address for the given
+// operator. If the staking provider has not been registered for the operator,
+// the returned address is empty and the boolean flag is set to false.
+// If the staking provider has been registered, the address is not empty and the
+// boolean flag indicates true.
+func (bc *BeaconChain) OperatorToStakingProvider(
+	operator chain.Address,
+) (chain.Address, bool, error) {
+	stakingProvider, err := bc.randomBeacon.OperatorToStakingProvider(
+		common.HexToAddress(operator.String()),
+	)
+	if err != nil {
+		return "", false, fmt.Errorf(
+			"failed to map operator %v to a staking provider: [%v]",
+			operator,
+			err,
+		)
+	}
+
+	if (stakingProvider == common.Address{}) {
```

Comment:
Done in 66f1efb9701d3f926053e8b35a65e78abc844f87.
It does work, and it's how `go-ethereum` compares empty addresses:
https://github.com/ethereum/go-ethereum/blob/d740d6e741562e98b3d75a98e7decdfee3bdd904/accounts/keystore/account_cache.go#L177

Alternatively, it could be:
```
if stakingProvider == (common.Address{}) {
   ...
}
```